### PR TITLE
free-courses-hi.md (adding resources)(changes made)

### PR DESCRIPTION
**Pull request by ravi0900**
## What does this PR do?
Add resources
### Description
added 2 hi courses
both courses have youtube playlist

### Why is this valuable (or not)?
game development course is very valuable as I watched, and dbms playlist is helpful for hindi native speaker who starting there jorney in dbms.

### How do we know it's really free?
youtube videos
### For book lists, is it a book? For course lists, is it a course? etc.
Course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up
Please add hacktoberfest-accepted label to this PR, thank you.
- Check the status of GitHub Actions and resolve any reported warnings!
ok

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
when you've made changes, please add `(changes made)` to the title of your PR
"Hacktoberfest" is not a useful PR title

**Comment by ravi0900** [comment]
@eshellman I'm sorry, I should not add so many hacktoberfest labels, but ya I really respect this project and your work, I humbly want to contribute here, I have made the changes, please accept my PR, Thank you.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -214,7 +214,8 @@
 ### Databases
 
 * [3.1 DBMS In Hindi (Complete Playlist)](https://www.youtube.com/playlist?list=PLmXKhU9FNesR1rSES7oLdJaNFgmuj0SYV) - Knowledge Gate
-* [Database Management System ( DBMS )](https://www.youtube.com/playlist?list=PLrjkTql3jnm-CLxHftqLgkrZbM8fUt0vn) - Education 4u
+* [Database Management System (DBMS)](https://www.youtube.com/playlist?list=PLrjkTql3jnm-CLxHftqLgkrZbM8fUt0vn) - Education 4u
+* [Database Management System (DBMS) in Hindi](https://www.youtube.com/watch?v=xy138_iUfaI&list=PLAOnhLRjMTMDigfUzaAAQo7lbfScPFtHs) - Jitendra Ajmedha, The Crazy Programmer
```
Comment (eshellman) [inline courses/free-courses-hi.md:218]: use the playlist! i.e. https://www.youtube.com/playlist?list=PLAOnhLRjMTMDigfUzaAAQo7lbfScPFtHs

**Review by eshellman — CHANGES_REQUESTED** [review]
also, use the instructors name OR the channel name, not both

**Pull request merged**